### DISCUSSION TITLE
Add portfolio comparation with S&P500

### DISCUSSION THREAD

**Discussion by rubengr** [ghostfolio/ghostfolio#3893] [Ideas]
Add portfolio comparation with S&P500

**Comment by morskipas** [comment]
Do you mean something like this or more detailed
![image](https://github.com/user-attachments/assets/29963c45-fbe8-4a0f-9dcc-75c2e7a3e457)

**Reply by rubengr** [reply]
Exactly! sorry don't know that this is already possible. Thanks

**Reply by swissbuechi** [reply]
Thank you.

How exactly can this be configured?

~~Overview / Markets just displayes "No data available" in my case.~~

Update: Found it!
